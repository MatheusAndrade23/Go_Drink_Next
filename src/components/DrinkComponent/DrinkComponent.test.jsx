import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { DrinkComponent } from '.';
import popularDrinks from '../../templates/Home/popular-drinks';

describe('<DrinkComponent />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <DrinkComponent drink={popularDrinks.drinks[1]} />,
    );
    expect(container).toMatchSnapshot();
  });
});
