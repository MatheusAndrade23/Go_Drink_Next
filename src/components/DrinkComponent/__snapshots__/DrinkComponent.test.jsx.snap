// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrinkComponent /> should render 1`] = `
.c0 {
  margin: 10px 0;
  cursor: pointer;
  height: 340px;
  width: 310px;
  border: 2px solid #393E46;
  border-radius: 10px;
  -webkit-transition: 0.5s;
  transition: 0.5s;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 h6 {
  height: 35px;
  margin-left: 22px;
  width: 260px;
  overflow: hidden;
  margin-top: 10px;
  text-overflow: ellipsis;
  white-space: nowrap;
}

.c0 img {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  margin-top: 20px;
  width: 260px;
  border: 2px solid #EEEEEE;
}

.c0:hover {
  box-shadow: 1px 0px 12px 0px rgba(255,255,255,0.64);
}

.c1 {
  color: #00ADB5;
  font-size: 24px;
  text-transform: none;
}

@media (max-width:340px) {
  .c0 {
    height: 290px;
    width: 260px;
  }

  .c0 h6 {
    width: 210px;
  }

  .c0 img {
    width: 210px;
  }
}

@media (max-width:270px) {
  .c0 {
    height: 260px;
    width: 230px;
  }

  .c0 h6 {
    width: 180px;
  }

  .c0 img {
    width: 180px;
  }
}

<div>
  <div
    class="c0"
    title="Margarita details"
  >
    <img
      alt="Margarita"
      loading="lazy"
      src="https://www.thecocktaildb.com/images/media/drink/5noda61589575158.jpg"
    />
    <h6
      class="c1"
    >
      Margarita
    </h6>
  </div>
</div>
`;
